content/en/docs/user-documentation/layout-builder/location-finder/_index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Location Finder
+description: A set of components to view and search YMCA locations.
+---
+
+{{< tabpane text=true >}}
+  {{< tab header="Video" lang="vid" >}}
+    {{< youtube  >}}
+  {{< /tab >}}
+  {{% tab header="Diagram" lang="pic" %}}
+![A screenshot of the Locations block filters and map.](lb-location-finder--filters-map.png)
+![A screenshot of the Locations block listing.](lb-location-finder--listing.png)
+  {{% /tab %}}
+{{< /tabpane >}}
+
+**Designs:** [Mobile & Desktop](<../../../../../../assets/img/designs/lb-ui-kit/Locations.jpg>)
+
+The **Location Finder** block provides search, filters, a map, and a listing your YMCA locations.
+
+## Amenities filters
+
+![A screenshot of the Location Finder's "filter by amenities" section.](lb-location-finder--amenities.png)
+
+**Location Finder** also now supports hierarchical amenities. That means you can arrange your list of amenities into categories instead of a simple alphabetical list.
+
+The Amenities taxonomy is managed at **Administration** > **Structure** > **Taxonomy** > **Amenities**. See [Taxonomy, Vocabularies, and Terms](/docs/user-documentation/taxonomy) for more info on managing Vocabularies.
+
+### Single-level amenities
+
+![A screenshot showing the Amenities taxonomy administration on the left and the filters display on the right.](lb-location-finder--flat-amenities.png)
+
+If you leave the Amenities terms in a flat list on their configuration page, the Location Finder filters will display according to their configured weights. Drag terms up or down in the list to rearrange them in the filters.
+
+### Hierarchical (parent/child) amenities
+
+![A screenshot showing a parent-child relationship in the Location finder filters.](lb-location-finder--parent-amenities.png)
+
+Y's with many amenities may choose to group them in categories. Once any Amenities term is nested, the Location Finder filters switch to a hierarchical display.
+
+To nest terms:
+
+- Go to the Amenities administration page.
```

Comment:
Can we have a link?

Reply:
Fixed in https://github.com/YCloudYUSA/yusaopeny_docs/commit/b608cd335f154ab57cb96b75331e91d61b46013b